package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "wagtail.org",
+  "version": "1.0.0",
+  "description": "This is the source code to [Wagtail's website](https://wagtail.org)",
+  "scripts": {
+    "//[ Linting & formatting ]//": "",
+    "lint:css": "stylelint --report-needless-disables './wagtailio/static/sass'",
+    "lint:js": "eslint --ext .js,.ts,.tsx --report-unused-disable-directives .",
+    "lint:format": "prettier --check '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "lint": "npm run lint:format && npm run lint:js && npm run lint:css",
+    "format": "prettier --write '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "//[ Build tasks ]//": "",
+    "build": "webpack --mode development --progress",
+    "build:prod": "webpack --mode production",
+    "start": "webpack --mode development --progress --watch",
+    "start:reload": "webpack-dev-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wagtail/wagtail.org.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/wagtail/wagtail.org/issues"
+  },
+  "homepage": "https://github.com/wagtail/wagtail.org#readme",
```

Here’s Wagtail’s `browserslist`:

```suggestion
  "homepage": "https://github.com/wagtail/wagtail.org#readme",
  "browserslist": [
    "Firefox ESR",
    "last 2 Chrome versions",
    "last 2 ChromeAndroid versions",
    "last 2 Edge versions",
    "last 1 Firefox version",
    "last 2 iOS versions",
    "last 3 Safari versions",
    "not safari 13",
    "not ie 11"
  ],
```

---

Thanks. Added.